web/src/features/hfiCalculator/components/PrepLevelCell.test.tsx

```diff
@@ -28,35 +27,35 @@ describe('PrepLevelCell', () => {
     expect(cell.className).toMatch(/makeStyles-prepLevel1-/)
     expect(cell.innerHTML).toBe('1')
   })
-  it('should return a cell with a classname of prepLevel1 and a text prep level of 1', () => {
-    const { getByTestId } = renderPrepLevel(2)
-    const cell = getByTestId('weekly-prep-level-afton')
-    expect(cell.className).toMatch(/makeStyles-prepLevel1-/)
-    expect(cell.innerHTML).toBe('1')
-  })
   it('should return a cell with a classname of prepLevel2 and a text prep level of 2', () => {
-    const { getByTestId } = renderPrepLevel(3)
+    const { getByTestId } = renderPrepLevel(2)
     const cell = getByTestId('weekly-prep-level-afton')
     expect(cell.className).toMatch(/makeStyles-prepLevel2-/)
     expect(cell.innerHTML).toBe('2')
   })
   it('should return a cell with a classname of prepLevel3 and a text prep level of 3', () => {
-    const { getByTestId } = renderPrepLevel(4)
+    const { getByTestId } = renderPrepLevel(3)
     const cell = getByTestId('weekly-prep-level-afton')
     expect(cell.className).toMatch(/makeStyles-prepLevel3-/)
     expect(cell.innerHTML).toBe('3')
   })
   it('should return a cell with a classname of prepLevel4 and a text prep level of 4', () => {
-    const { getByTestId } = renderPrepLevel(5)
+    const { getByTestId } = renderPrepLevel(4)
     const cell = getByTestId('weekly-prep-level-afton')
     expect(cell.className).toMatch(/makeStyles-prepLevel4-/)
     expect(cell.innerHTML).toBe('4')
   })
-  it('should return a cell with a classname of prepLevel4 and a text prep level of 4', () => {
+  it('should return a cell with a classname of prepLevel5 and a text prep level of 5', () => {
+    const { getByTestId } = renderPrepLevel(5)
+    const cell = getByTestId('weekly-prep-level-afton')
+    expect(cell.className).toMatch(/makeStyles-prepLevel5-/)
+    expect(cell.innerHTML).toBe('5')
+  })
+  it('should return a cell with a classname of prepLevel6 and a text prep level of 6', () => {
     const { getByTestId } = renderPrepLevel(6)
     const cell = getByTestId('weekly-prep-level-afton')
-    expect(cell.className).toMatch(/makeStyles-prepLevel4-/)
-    expect(cell.innerHTML).toBe('4')
+    expect(cell.className).toMatch(/makeStyles-prepLevel6-/)
```

Comment:
Why did these all move down 1?

Reply:
I just realized they lined up wrong; was looking at the wrong line numbers (this makes sense still nothing changed)

Reply:
We pass in the `prepLevel` into the `PrepLevelCell` component instead of the `meanIntensity` now to simplify. This component is just responsible for styling the inpurt.